src/test/java/teammates/test/util/PriorityInterceptor.java

```diff
@@ -70,16 +70,6 @@ private String getClassName(IMethodInstance mi) {
             }
             
             private int packagePriorityOffset(String packageName) {
-                //Storage tests go first!
-                if (packageName.contains("storage")) {
-                    return 1000000;
-                }
-                //Action tests go last. (in component tests)
-                if (packageName.contains("teammates.test.cases.ui")) {
-                    return -1000000;
-                }
-
-
                 int index = packageOrder.indexOf(packageName);
 
                 if (index == -1) {
```

Comment:
Copying from the class documentation:
```java
/**
 * This class prioritizes methods based on the following:
 *      1) Orders methods based on package name as ordered/found in testng.xml
 *      2) Orders methods based on package name in lexical order
 *      3) Orders methods by class priority e.g. Add "@Priority(1)" to class
 *      4) Orders methods by class name in lexical order
 *      5) Orders methods by priority e.g. Add "@Priority(1)" to method
 */
```
This method is part of (1) and `index == -1` implies that the package name cannot be found, thus moving on to (2).